src/app/http.ts

```diff
@@ -0,0 +1,62 @@
+import fetch from 'node-fetch';
+
+const host = process.env.MM_URL || 'http://localhost:8065';
+const pathPrefix = `${host}/plugins/com.mattermost.apps`;
+
+const pathInstall = `${pathPrefix}/install`;
+const pathSubscribe = `${pathPrefix}/subscribe`;
+const pathConnectedInstall = `${pathPrefix}/connected_install`;
+
+// http constains methods for handling requests to and from the
+// mattermost-plugin-apps API
+class MattermostPluginHTTP {
```

Comment:
Looks like this file isn't being imported anywhere

Reply:
removed.